packages/tools/pie-icons-configs/README.md

```diff
@@ -40,4 +40,4 @@ You can run the tests with `yarn test`.
 
 ## Contributing
 
-Before starting please read our [contributing guide](https://vue.pie.design/?path=/story/documentation-getting-started-contributing--page)
+Before starting please read our [contributing guide](https://pie.design/engineers/contributing/)
```

Comment:
While you're here can we update the test script above to `yarn test --filter=pie-icons-configs`?